src/lib/swapper/swappers/ThorchainSwapper/evm/routerCallData.test.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment node
+ */
+
```

Comment:
The guts of making the tests work. Failure to do this will result in large problems, see https://github.com/jestjs/jest/issues/2549